server/src/main/java/org/elasticsearch/search/aggregations/pipeline/BucketHelpers.java

```diff
@@ -41,9 +41,40 @@
      *
      * "insert_zeros": empty buckets will be filled with zeros for all metrics
      * "skip": empty buckets will simply be ignored
+     * "none": for empty buckets the values provided by the metrics will still be used if they are available
```

Comment:
```suggestion
     * "keep_values": for empty buckets the values provided by the metrics will still be used if they are available
```
Also, we could make these `{@link }` tags, then renaming would catch them.

Reply:
"none" was never released, it is a leftover from my earlier iteration.